### ISSUE TITLE
build.sh fails on Mac with java.util.zip.ZipException: duplicate entry: org/mockito/captor/Captor.class

### ISSUE THREAD

**Issue by mockitoguy** [mockito/mockito-scala#395]
Currently, the build fails on my Mac with `java.util.zip.ZipException: duplicate entry: org/mockito/captor/Captor.class`

The full stack trace: https://gist.github.com/mockitoguy/462a1bea005ce40e563d020e42f9d793

After the failure, I see the following Captor.class files:

```
mockito-scala$ find . -name Captor.class
./core/target/scala-2.11/classes/org/mockito/captor/Captor.class
./core/target/scala-2.12/classes/org/mockito/captor/Captor.class
./macro/target/scala-2.11/classes/org/mockito/captor/Captor.class
./macro/target/scala-2.13/classes/org/mockito/captor/Captor.class
./macro/target/scala-2.12/classes/org/mockito/captor/Captor.class
```

@bbonanno, can you help with this? Can you configure the SBT build so that it deals with those duplicates? Otherwise it's difficult for me to iterate on the Maven Central releases (https://github.com/mockito/mockito-scala/issues/394)

**Comment by ultrasecreth** [comment]
Sure, I'll try to take a look tomorrow

**Comment by ultrasecreth** [comment]
Hi @mockitoguy, just ran the build on my Mac and worked like charm :(

The thing I noticed, based on the gist, is that the version is trying to package (`mockito-scala_2.11-0.4.7.jar`) is a very old one, mine is trying to build `mockito-scala_2.11-1.16-SNAPSHOT.jar`, which makes more sense as the last released version to maven central was `1.16.37`. Could it be you're working on `master` and not `release/1.x`?

**Comment by mockitoguy** [comment]
Oups, that works :D
